src/io/ioctl.rs

```diff
@@ -42,7 +45,12 @@ pub fn ioctl_fioclex<Fd: AsFd>(fd: Fd) -> io::Result<()> {
 #[inline]
 #[doc(alias = "FIONBIO")]
 pub fn ioctl_fionbio<Fd: AsFd>(fd: Fd, value: bool) -> io::Result<()> {
-    backend::io::syscalls::ioctl_fionbio(fd.as_fd(), value)
+    // SAFETY: FIONBIO is a pointer setter opcode.
+    #[allow(unsafe_code)]
```

Comment:
Could you move these `#[allow(unsafe_code)]`s to a `#![allow(unsafe_code)]` at the top of the file?

The goal isn't to add extra ceremony to every `unsafe` block, but to try to encourage factoring out `unsafe` blocks from files that don't otherwise have any `unsafe` in them.